docs/docs-content/enterprise-version/upgrade/upgrade-k8s/airgap.md

```diff
@@ -0,0 +1,432 @@
+
+4.  Download the airgap setup binary using the credentials our support team provided along with the binary version. Use
+    the following command template to set up a `VERSION` variable you'll use in the following steps.
+
+    ```shell
+    export VERSION=<x.y.z>
+    ```
+
+    Consider the following example for reference.
+
+    ```shell
+    export VERSION=4.2.7
+    ```
+
+5.  Use the following command template to download the airgap setup binary.
+
+    ```shell
+    curl --user <username>:<password> https://software-private.spectrocloud.com/airgap/$VERSION/airgap-v$VERSION.bin  \
+    --output airgap-v$VERSION.bin
+    ```
+
+6.  Use the following command to make the airgap binary executable.
+
+    ```shell
+    chmod +x airgap-v$VERSION.bin
+    ```
+
+7.  Use the following command to start the airgap setup binary.
+
+    ```shell
+    ./airgap-v$VERSION.bin
+    ```
+
+    After the binary execution is done, you should receive the following success message (the example is shortened for
+    brevity). As a result of running the binary, you should have a `spectro-manifests` archive in your temporary
+    directory.
+
+    ```shell
+    Verifying archive integrity...  100%   MD5 checksums are OK. All good.
+    Uncompressing Airgap Setup - Version 4.0.17  100%
+    Setting up Packs
+    - Pushing Pack cni-calico:3.25.1
+    ...
+    Setting up Images
+    - Pushing image docker.io/kindest/kindnetd:v20230227-15197099
+    - Pushing image gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.22.8
+    ...
+    // highlight-start
+    Preparing Manifests Archive
+    Manifests are available in /tmp/spectro-manifests-1696971110.zip. Extract the archive to a file server to serve as a Spectro Cloud Repository
+    // highlight-end
+    Setup Completed
+    ```
+
+8.  Move the `spectro-manifests` archive to a directory that your file server can access and use the following command
+    template to unzip it.
+
+    ```shell
+    unzip spectro-manifests-<file-id>.zip -d /target/folder
+    ```
+
+9.  Refer to the [Additional Packs](../../install-palette/airgap/supplemental-packs.md) page and update the packages you
+    are currently using. You must update each package separately.
+
+    To update a package, use the following command template to download and execute the pack binary.
+
+    ```shell
+    curl --remote-name <pack-name-url> && chmod +x <pack-name-version>.bin && ./<pack-name-version>.bin
+    ```
+
+    Consider the following example for reference.
+
+    ```shell
+    curl --remote-name https://software-private.spectrocloud.com/airgap/packs/airgap-pack-aws-alb-2.5.1.bin \
+      && chmod +x airgap-pack-aws-alb-2.5.1.bin \
+      && ./airgap-pack-aws-alb-2.5.1.bin
+    ```
+
+    ```shell
+    Verifying archive integrity...  100%   MD5 checksums are OK. All good.
+    Uncompressing Airgap Pack - aws-alb Version 4.0.17  100%
+    Setting up Packs
+    - Pushing Pack aws-alb:2.5.1
+    Setting up Images
+    Setup Completed
+    ```
```

Comment:
I don't think we need to duplicate the information we provide in the additional packs page.